Sources/Jobs/JobsCommand.swift

```diff
@@ -66,68 +84,61 @@ public class JobsCommand: Command {
         signal(SIGINT, SIG_IGN)
         intSignalSource.resume()
         
+        
         var shutdownPromises: [EventLoopPromise<Void>] = []
-        for eventLoop in elg.makeIterator()! {
-            let sub = context.container.subContainer(on: eventLoop)
+        for eventLoop in elg.makeIterator() {
             let console = context.console
-            let queueName = context.options["queue"] ?? QueueName.default.name
-            let shutdownPromise: EventLoopPromise<Void> = eventLoop.newPromise()
+            let shutdownPromise: EventLoopPromise<Void> = eventLoop.makePromise()
             
             shutdownPromises.append(shutdownPromise)
             
             eventLoop.submit {
                 try self.setupTask(eventLoop: eventLoop,
-                                   container: sub,
                                    queueName: queueName,
                                    console: console,
                                    promise: shutdownPromise)
-            }.catch {
+            }.whenFailure {
                 console.error("Could not boot EventLoop: \($0)")
             }
         }
         
-        return .andAll(shutdownPromises.map { $0.futureResult }, eventLoop: elg.next())
+        try EventLoopFuture.andAllComplete(shutdownPromises.map { $0.futureResult }, on: elg.next()).wait()
     }
     
     private func setupTask(eventLoop: EventLoop,
-                           container: SubContainer,
                            queueName: String,
-                           console: Console,
+                           console: ConsoleKit.Console,
                            promise: EventLoopPromise<Void>) throws
     {
         let queue = QueueName(name: queueName)
-        let queueService = try container.make(QueueService.self)
-        let jobContext = try container.make(JobContext.self)
         let key = queue.makeKey(with: queueService.persistenceKey)
-        let config = try container.make(JobsConfig.self)
-        
-        _ = eventLoop.scheduleRepeatedTask(initialDelay: .seconds(0), delay: queueService.refreshInterval) { task -> EventLoopFuture<Void> in
+        _ = eventLoop.scheduleRepeatedAsyncTask(initialDelay: .seconds(0), delay: queueService.refreshInterval) { task -> EventLoopFuture<Void> in
```

Comment:
while we're here, should we try to handle this future better? the least we could do is print something if it fails